CS483/CS483/Kartaclysm/Services/IO/InputActionMapping.cpp

```diff
@@ -400,7 +470,7 @@ namespace Kartaclysm
 #ifdef _DEBUG
 				printf("ControlBindings.xml error: restoring defaults");
 #endif
-				ResetUserControlBindings(it->first == Input::eKeyboard, it->first == Input::eJoystick);
+				ResetUserControlBindings(it->first == Input::eKeyboard1, it->first == Input::eJoystick);
```

Comment:
The default values and resetting to KeyBoard1 are okay for testing purposes because the other keyboards are loaded from XML. If we want a more permanent solution, we'll have to remember to change this.